.github/workflows/post-merge-deploy.yml

```
@@ -0,0 +1,48 @@
+name: Package for Deployment
```

```suggestion
name: Package for deployment
```
Consistent pattern with other workflows